miriway.cpp

```diff
@@ -66,27 +67,47 @@ std::map<std::string, ShellCommands::CmdFunctor> const wm_command =
         { "exit", [](ShellCommands* sc, bool shift) { sc->exit(shift); } },
     };
 
+struct ShellComponentRunInfo
+{
+    time_t last_run_time = 0;
```

Comment:
I agree with @RAOF - using `std::chrono` seems more natural

Reply:
In this case, I feel `std::time` is much easier to read/understand. I know that I want the time in epoch seconds. `std::chrono` is feeling a little overkill IMO